java/core/src/java/org/apache/orc/impl/RecordReaderImpl.java

```diff
@@ -110,10 +111,9 @@ static int findColumns(SchemaEvolution evolution,
       TypeDescription fileColumn = evolution.getFileType(readerColumn);
       return fileColumn == null ? -1 : fileColumn.getId();
     } catch (IllegalArgumentException e) {
-      if (LOG.isDebugEnabled()){
-        LOG.debug("{}", e.getMessage());
-      }
-      return -1;
+      throw new IllegalArgumentException("Filter could not find column with name: " +
```

Comment:
Style is a bit off here?

Reply:
@pgaref what are you asking for?

Reply:
Looked like the indentation was off here (on my IDE at least) but just realised it was just split in 2 lines instead of 3 -- ignore the comment please